### ISSUE TITLE
[Bug] Accessibility - Cannot hover over tooltip content

### ISSUE THREAD

**Issue by meganthecoder** [apache/echarts#21422]
### Version

6.0.0

### Link to Minimal Reproduction

https://codepen.io/meganthomas/pen/gbryOGo

### Steps to Reproduce

1. Hover your mouse over the Search Engine (light blue) data point for Mon. The tooltip should appear.
2. Try to hover your mouse over the content in the tooltip.
3. Notice how the tooltip keeps moving so it's impossible to hover over it. The content of the tooltip will change to the other days (Tues, Weds, etc.) until your mouse moves off of the chart and the tooltip disappears.

### Current Behavior

It's impossible to hover over the tooltip with a mouse. This violates WCAG Success Criteria 1.4.13: Content on Hover or Focus. According to this accessibility criteria, a pointer must be able to move over the tooltip without it disappearing.

More info: https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus.html

### Expected Behavior

A mouse should be able to hover over the tooltip.

### Environment

```markdown
- OS: macOS Sequoia
- Browser: Chrome 142.0.7444.177
- Framework: N/A
```

**Comment by helgasoft** [comment]
```enterable:true``` ?
